From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@julesman101__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@julesman101 Thanks for the contribution!  I'll have the author review the change.

KBDAzure commented:
Hi Julesman101 -- 
Thanks for the input and the correction. This content was recently moved from MSDN and I missed that update. I'll update the article today to cite 1023 GB to match what's documented in the 'Sizes for Virtual Machines' page (https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-size-specs/).

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

v-prhal committed:
Merge pull request #2 from MicrosoftDocs/master

Sync with master

v-prhal committed:
Bug 1123567 - Landing page update due to TOC update

v-prhal committed:
Bug 1123567

v-prhal committed:
Bug 1123567 - Landing page update due to TOC update

v-prhal committed:
Bug 1123567 - Landing page update due to TOC update

v-prhal committed:
Bug 1123567 - Landing page update due to TOC update

v-prhal committed:
Bug 1123567 - Landing page update due to TOC update

v-prhal committed:
Bug 1123567 - Landing page update due to TOC update

v-prhal committed:
Bug 1123567 - Landing page update due to TOC update

v-prhal committed:
Bug 1123567 - Landing page update due to TOC update

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-656

peterpogorski committed:
Specify how to build dotnet core applications on Jenkins

richagi committed:
Notice that SDK is deprecated

richagi committed:
Remove references to SDK

BryanLa committed:
updates

BryanLa committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into bryanla-msi-tutorial-prereq

richagi committed:
Remove two-way SMS references

richagi committed:
editing date to nov 14 instead of 15

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

v-nagta committed:
USER STORY 1129479 SaaS App Tutorial: Teamwork update

BryanLa committed:
tutorial prereqs

Commit:
linux cluster on windows updates

CamSoper committed:
Major revisions

CamSoper committed:
Acrolinx suggestions

CamSoper committed:
dealt with warning

CamSoper committed:
Missing period

cephalin committed:
remove classic portal reference

MGoedtel committed:
updated article for new version

cephalin committed:
acrolinx

cephalin committed:
acrolinx

cephalin committed:
acrolinx

richagi committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into richagifraud

bandersmsft committed:
cost mgt - updating manage costs for tags

richagi committed:
Edits to deprecation announcement. Added detail for two-way SMS. Acrolynx fixes.

richagi committed:
Note that two-way SMS is deprecated instead of removing reference.

bandersmsft committed:
updating faq

bandersmsft committed:
fixed typo

bandersmsft committed:
fixed typo

anumjs committed:
added -mt for app link

bandersmsft committed:
fixing typo

Commit:
reviewing stale topics

Commit:
minor edit to TOC

kgremban committed:
Review of C# tutorial

Commit:
Updated the details on RU size limits for encoding source files

Commit:
acrolinx edits

kgremban committed:
Images

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-656

Commit:
more acrolinx edits

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into nov21-csharpreview

kgremban committed:
Olivier's edits

kgremban committed:
Acrolinx

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

CamSoper committed:
feedback

CamSoper committed:
spelling

Commit:
task 656: incorporate review comments

Commit:
task 656: incorporate review comments

v-nagta committed:
USER STORY 1132141 SaaS App Tutorial: Cezanne HR Software Tutorial Update

MGoedtel committed:
updated several items to improve readability

cherylmc committed:
Update vpn-gateway-about-vpn-devices.md

MGoedtel committed:
updated items

bandersmsft committed:
updated regions in TOC

cherylmc committed:
Update how-to-npm.md

MGoedtel committed:
updated items

tdykstra committed:
create include - app settings to local.settings

bandersmsft committed:
fixed PR review issues

BryanLa committed:
updates

carolinacmoravia committed:
M15127:  Possible type mistake: size

Hello, @iainfoulds, 
Translator has reported possible source content issue. It seems that it should say "side" instead of "size", according to the context. 
 
Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

MGoedtel committed:
Updated version details for prerequisites

peterpogorski committed:
Merge branch 'master' into jenkins-cicd-dotnet

georgewallace committed:
storage quickstart - correct formatting

anumjs committed:
Updated folder name in the doc 

Provision and Catalog to ProvisionTenants

Commit:
Merge pull request #28462 from MGoedtel/UpdateLAMalware1122

Updated version details for prerequisites

gsacavdm committed:
Minor fixes on gov-aad-auth-qs

Addressing feedback provided in PR #28280

PRMerger10 committed:
Merge pull request #28465 from georgewallace/patch-1

storage quickstart - correct formatting

daveirwin1 committed:
Added min SCOM version

gsacavdm committed:
Fix TOC title for gov-aad-auth-qs

Fixing as per PR #28280

v-shils committed:
Merge pull request #26241 from BryanLa/bryanla-msi-tutorial-prereq

Add prerequisite block

BryanLa committed:
final preqreqs update for tutorials

v-shils committed:
Merge pull request #28454 from tdykstra/include

create include - app settings to local.settings

PRMerger14 committed:
Merge pull request #28467 from gsacavdm/patch-5

Minor fixes on gov-aad-auth-qs

v-shils committed:
Merge pull request #28376 from kgremban/nov21-csharpreview

Review of C# tutorial

v-shils committed:
Merge pull request #28377 from johndeu/master

Updated the details on RU size limits for encoding source files

msmbaldwin committed:
Fixed h1s

MGoedtel committed:
Updated article per pr reviewer

PRMerger18 committed:
Merge pull request #28470 from daveirwin1/master

Added min SCOM version

Commit:
Merge pull request #28356 from richagi/richagifraud

Deprecation announcements

v-shils committed:
Merge pull request #28378 from bandersmsft/master

cost mgt - updating manage costs and faq

msmbaldwin committed:
Fixed h1s

cephalin committed:
PRmerger feedback

christiankuhtz committed:
tweaks

Commit:
task 656: edit fixes

anumjs committed:
added missing backslash

v-shils committed:
Merge pull request #28032 from peterpogorski/jenkins-cicd-dotnet

Specify how to build dotnet core applications on Jenkins

v-shils committed:
Merge pull request #27643 from v-prhal/master

Bug 1123567 - Landing page update due to TOC update

ktoliver committed:
Merge pull request #28301 from CamSoper/get-started-sf-actor

Reliable Actors - Revisions

PRMerger14 committed:
Merge pull request #28476 from msmbaldwin/fixh1s1

Fixed h1s

christiankuhtz committed:
tweaks

PRMerger16 committed:
Merge pull request #28477 from msmbaldwin/fixh1s2

Fixed h1s

ktoliver committed:
Merge pull request #28431 from v-nagta/cezanne

USER STORY 1132141 SaaS App Tutorial: Cezanne HR Software Tutorial Up…

christiankuhtz committed:
tweaks

christiankuhtz committed:
tweaks

christiankuhtz committed:
tweaks

ktoliver committed:
Merge pull request #28469 from gsacavdm/patch-6

Fix TOC title for gov-aad-auth-qs

christiankuhtz committed:
tweaks

Commit:
addressing review issues - split bullet and headings

ktoliver committed:
Merge pull request #28341 from MGoedtel/UpdateVMMgmt1121

updated article for new version

v-thepet committed:
fix broken links

v-shils committed:
Merge pull request #28232 from v-nagta/teamwork

USER STORY 1129479 SaaS App Tutorial: Teamwork update

v-shils committed:
Merge pull request #28472 from BryanLa/bryanla-msi-tutorial-prereq

final preqreqs update for tutorials

MikeDodaro committed:
Typo in code endpoint parameter

v-shils committed:
Merge pull request #28339 from cephalin/portal

remove class portal reference

PRMerger17 committed:
Merge pull request #28482 from v-thepet/newfix

production bug: fix broken links

kgremban committed:
Fixing regional note in ASA tutorial

PRMerger17 committed:
Merge pull request #28461 from carolinacmoravia/patch-470

M15127:  Possible type mistake: size

CamSoper committed:
Clarifying when to use

CamSoper committed:
Acrolinx

ktoliver committed:
Merge pull request #28478 from christiankuhtz/patch-37

tweaks

rwike77 committed:
Updating H1

PRMerger19 committed:
Merge pull request #28486 from kgremban/nov22-asanote

Fixing regional note in ASA tutorial

CamSoper committed:
URL paste fail

ktoliver committed:
Merge pull request #28416 from dksimpson/dean-656

Dean 656

Commit:
Merge pull request #28369 from anumjs/master

added -mt for app link

kmouss committed:
Updates per CSS feedback

Updates per CSS feedback and new public link for AHB compliance

GitHubber17 committed:
Merge pull request #28372 from bradsev/update-stale-topics-nov2017

reviewing stale topics

kmouss committed:
Fix typos

Fix typos

kmouss committed:
fix lint warning

fix lint warning

CamSoper committed:
team feedback

PRMerger17 committed:
Merge pull request #28491 from rwike77/patch-28

Updating H1

rwike77 committed:
Updating H1

PRMerger12 committed:
Merge pull request #28488 from CamSoper/azure-files

Clarifying when to use WindowsAzure.Storage

Ja-Dunn committed:
Merge pull request #28296 from suhuruli/master

linux cluster on windows updates

PRMerger16 committed:
Merge pull request #28492 from kmouss/master

Updates for new AHB compliance link and CSS feedback

PRMerger12 committed:
Merge pull request #28493 from rwike77/patch-29

Updating H1

PRMerger18 committed:
Merge pull request #28484 from MikeDodaro/fix-typo-video-insights

Typo in code endpoint parameter needs to be fixed

PRMerger16 committed:
Merge pull request #28446 from cherylmc/patch-3

Update vpn-gateway-about-vpn-devices.md

PRMerger16 committed:
Merge pull request #28448 from cherylmc/patch-4

Update how-to-npm.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge pull request #28495 from MicrosoftDocs/FromPublicRepo

From public repo
